README.md

```diff
@@ -97,137 +74,129 @@ After you're finished please remove all the comments and instructions!
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
-<!-- LIVE DEMO -->
+<!-- ER DIAGRAM-->
+## :card_index: Entity Relationship Diagram <a name="er-diagram"></a>
+![erd](https://user-images.githubusercontent.com/54780027/231417553-6cf720cb-76db-47f9-8923-3b7c8af175f9.PNG)
 
-## 🚀 Live Demo <a name="live-demo"></a>
+<!-- React Frontend -->
+## <img src="https://upload.wikimedia.org/wikipedia/commons/thumb/a/a7/React-icon.svg/539px-React-icon.svg.png" width="23" height="20"/> React Frontend <a name="react-frontend"></a>
+- Here is the Frontend part of the project [Front End](https://github.com/PromzzyKoncepts/Book-Appointment-Front-end)
+
+## 🚀 kanban board <a name="board"></a>
 
-> Add a link to your deployed project.
+- Here is the kanban board of the project
+- ![kanban board](https://user-images.githubusercontent.com/54780027/231417798-2c94099c-f01a-478a-b9c4-3f76d9276478.PNG)
 
-- [Live Demo Link](https://google.com)
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- [Live Demo :rocket:](https://regal.netlify.app/) :smiley:
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
 <!-- GETTING STARTED -->
-
 ## 💻 Getting Started <a name="getting-started"></a>
 
-> Describe how a new developer could make use of your project.
-
 To get a local copy up and running, follow these steps.
-
 ### Prerequisites
 
-In order to run this project you need:
-
-<!--
-Example command:
-
-```sh
+To run this project you need:
+`Git` and `Ruby`
+```
  gem install rails
 ```
- -->
 
-### Setup
-
-Clone this repository to your desired folder:
+### Install
 
-<!--
-Example commands:
+Install this project with:
 
 ```sh
-  cd my-folder
-  git clone git@github.com:myaccount/my-project.git
+  bundle install
+```
+- Generate a secret key using `rails secret`
+- Create a  `.env` file with the following content:
+```
+POSTGRES_USER=postgres
+# If you declared a password when creating the database:
+POSTGRES_PASSWORD=YourPassword
+POSTGRES_HOST=localhost
+POSTGRES_DB=Hello_Rails_Backend_development
+POSTGRES_TEST_DB=Hello_Rails_Backend_test
+# Devise secret key
+DEVISE_JWT_SECRET_KEY=Secret Key you generated
 ```
---->
 
-### Install
+Create a Database (Mandatory)
 
-Install this project with:
+```
+rails db:create
+```
 
-<!--
-Example command:
+If you have made a migration then run this command
 
-```sh
-  cd my-project
-  gem install
 ```
---->
-
+rails db:migrate
+```
+```
+rails db:seed
+```
 ### Usage
 
 To run the project, execute the following command:
 
-<!--
-Example command:
-
 ```sh
-  rails server
+   rails s -p 3001 # # for serving the API on localhost:3001
 ```
---->
-
-### Run tests
-
-To run tests, run the following command:
-
-<!--
-Example command:
 
-```sh
-  bin/rails test test/models/article_test.rb
+Run tests
 ```
---->
-
-### Deployment
-
-You can deploy this project using:
-
-<!--
-Example:
-
-```sh
-
+bundle exec rspec
```

Comment:
- [ ] When I run the tests according to your instructions, I get this:

![image](https://user-images.githubusercontent.com/57421823/231534220-c1dc6172-d99d-4ea2-9b77-6b9cd580ff5d.png)

## Explanation

This is because some of the tests are swagger tests.

## Tips and extra help

Just be more specific with your commands in your instructions:

```bash
 bundle exec rspec ./spec/models
 bundle exec rspec ./spec/routes
```

That way, your users will be able to run the tests correctly, and make them pass.  💪 

- [ ] When I run `bundle exec rspec ./spec/models`, I get a failed test on my end:

![image](https://user-images.githubusercontent.com/57421823/231534999-9289d9fe-0ddd-4f5d-89e6-7f346b59870e.png)

It seems that the test on line 56 is not passing.  Kindly take a look at that one.  If there are special procedures to make this one pass, kindly include the instructions on your readme. 👍